packages/kamino-sdk/tests/stratCreation.test.ts

```diff
@@ -432,6 +431,15 @@ describe('Kamino strategy creation SDK Tests', () => {
     console.log('orca positions count', positionsCount);
   });
 
+  it.skip('get raydium positions for pool', async () => {
```

Comment:
Why skip?

Reply:
the mainnet public RPC is flaky so it will be flaky as well, I added it to test locally

Reply:
the mainnet public RPC is flaky so it will be flaky as well, I added it to test locally